### DISCUSSION TITLE
Good collab with Hypercomic. Check my badges and show yours!

### DISCUSSION THREAD

**Discussion by FrondEnt** [zkSync-Community-Hub/zksync-developers#350] [Show and Tell]
### Project Name

Hypercomic

### Repo Link

https://hypercomic.io/zk24/collect_hk.html

### Technologies Used

exclusive ZK24 NFTs!

### Describe your project. What does it do? Why did you create it? Any interesting challenges you faced?

![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/109918389/c9930b90-3d8e-4ace-81d0-40fae5d1da89)

### What specific feedback are you looking for?

Go check who have more badges!:)

### How can the community help or contribute to your project?

Collect Badges from Hypercomic and zkSync collaboration!

**Comment by butthurt-codewriter** [comment]
![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/109068934/df8d2fac-a139-42a1-97fc-2944c2f0a982)

This is mine

**Reply by Michael1Jordan1** [reply]
wow great work :) I have same results 
![Screenshot_44](https://github.com/zkSync-Community-Hub/zksync-developers/assets/88579393/af5761af-3097-49c0-b426-d0231fe8e14c)

**Reply by Grakety** [reply]
I also have the same badges!
![zkSync badges](https://github.com/zkSync-Community-Hub/zksync-developers/assets/116459522/12b68f5b-ab23-4868-96c2-0593030ad7e8)

**Comment by evmparser** [comment]
The Hypercomic project utilizes exclusive ZK24 NFTs (Non-Fungible Tokens). It likely serves as a platform for creating, collecting, and potentially trading these unique digital assets. Challenges may include ensuring platform security, scalability, and community engagement.

**Comment by noblepeter2000** [comment]
You're doing well. 
![Uploading Screenshot_2024-02-19-17-13-56-76_1a01c1dfc8c0ea31a9d50a905953499e.jpg…]()

**Comment by AndreyP55** [comment]
I also have the same badges!
![11111111111111111](https://github.com/zkSync-Community-Hub/zksync-developers/assets/137103043/5f703b51-5db3-4147-9733-491e2d22790f)

**Comment by velezlia** [comment]
Project Name: Hypercomic

Repo Link: [Hypercomic Repo](https://hypercomic.io/zk24/collect_hk.html)

Technologies Used: Exclusive ZK24 NFTs

Description:
Hypercomic is a project that offers exclusive ZK24 NFTs. These NFTs are unique digital collectibles created in collaboration with zkSync. Users can collect badges from Hypercomic and zkSync collaboration, each representing a unique achievement or milestone.

Specific Feedback:
We invite the community to explore Hypercomic and discover who has collected more badges. Your feedback on the user experience, badge design, and overall concept is greatly appreciated.

How to Contribute:
You can contribute to Hypercomic by collecting badges from the collaboration and providing feedback on the project's functionality and design. Additionally, sharing your experience with others and spreading the word about Hypercomic would be valuable contributions to the project's growth.

Additional Information:
No further information provided.

**Comment by wunrorobudaniel** [comment]
Hypercomic is a groundbreaking project leveraging exclusive ZK24 NFTs! Explore the collection at https://hypercomic.io/zk24/collect_hk.html. Our aim is to gamify NFT ownership with badges. Seek out who has more badges and join the adventure! Contributions from the community are welcome in collecting badges and exploring the Hypercomic universe.

**Comment by sarahzitzner** [comment]
Exciting collaboration between Hypercomic and zkSync! Looking forward to exploring the exclusive ZK24 NFTs. Keep up the great work!

**Comment by deanhobson** [comment]
Hypercomic is a project that utilizes exclusive ZK24 NFTs. It allows users to collect badges from the Hypercomic and zkSync collaboration. The project is seeking feedback on checking who has more badges, and the community can contribute by collecting badges from the collaboration.

**Comment by digijoe7** [comment]
<img width="1089" alt="Screenshot 2024-04-10 at 12 21 20 PM" src="https://github.com/zkSync-Community-Hub/zksync-developers/assets/138716885/a009bef3-97e9-43ad-8757-2fef7bfe3e7a">
Stacked
